best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,80 @@
+The best practice recommends not to use more than 2 properties, `production` and `apiUrl`.
+To achieve that, we need to make some changes on environment files as you can see below:
+```
+https://medium.com/most-wanted-experts/angular-application-configurations-best-practices-2e28514b9b4e
+```
+
+The configuration on `angular.json` does not make effect when we buld the application in NativeScript. Below we have a work around to use environment.
```

Comment:
You are referencing `angular.json` is this README focused on Angular?

Reply:
NativeScript-Angular projects. I modify the title to show that info.